lang/python/python-cryptodome/patches/002-omit-tests.patch

```diff
@@ -17,34 +17,34 @@
  ]
  package_dir = {"Crypto": "lib/Crypto"}
  package_data = {
-@@ -268,30 +258,6 @@
+@@ -268,30 +258,6 @@ package_data = {
      "Crypto.Signature" : [ "*.pyi" ],
      "Crypto.IO" : [ "*.pyi" ],
      "Crypto.Util" : [ "*.pyi" ],
 -    "Crypto.SelfTest.Cipher" : [
--        "test_vectors/AES/*.rsp",
--        "test_vectors/TDES/*.rsp",
--        "test_vectors/wycheproof/*.json",
+-        "test_vectors/AES/*.*",
+-        "test_vectors/TDES/*.*",
+-        "test_vectors/wycheproof/*.*",
 -    ],
 -    "Crypto.SelfTest.Hash" : [
--        "test_vectors/SHA1/*.rsp",
--        "test_vectors/SHA2/*.rsp",
--        "test_vectors/SHA3/*.txt",
--        "test_vectors/keccak/*.txt",
--        "test_vectors/BLAKE2s/*.txt",
--        "test_vectors/BLAKE2b/*.txt",
--        "test_vectors/wycheproof/*.json",
+-        "test_vectors/SHA1/*.*",
+-        "test_vectors/SHA2/*.*",
+-        "test_vectors/SHA3/*.*",
+-        "test_vectors/keccak/*.*",
+-        "test_vectors/BLAKE2s/*.*",
+-        "test_vectors/BLAKE2b/*.*",
+-        "test_vectors/wycheproof/*.*",
 -    ],
 -    "Crypto.SelfTest.Signature" : [
 -        "test_vectors/DSA/*.*",
 -        "test_vectors/ECDSA/*.*",
 -        "test_vectors/PKCS1-v1.5/*.*",
 -        "test_vectors/PKCS1-PSS/*.*",
--        "test_vectors/wycheproof/*.json",
+-        "test_vectors/wycheproof/*.*",
 -    ],
 -    "Crypto.SelfTest.PublicKey" : [
 -        "test_vectors/ECC/*.*",
 -    ],
  }
```

Comment:
i'm curios: is there a way to patch the setup script to omit tests without this big patch ?

i.e. add a new line that nullifies the entries ?
something like:
```
package_data.pop["Crypto.SelfTest.PublicKey"]
```
?

Reply:
```
packages = [i for i in packages if not i.startswith('Crypto.SelfTest')]
package_data = {k: v for k, v in package_data.items() if not k.startswith('Crypto.SelfTest')}
```
This works with Python 2 & 3.

PS: Should I add an option to select whether to exclude tests?

Reply:
> PS: Should I add an option to select whether to exclude tests?

You are the maintainer :)
You decide.

My particular preference was to reduce the patch size.
Smaller patches are easier to maintain after a few upgrades.

Reply:
I decided not to add that option because no packages provides such option.

Reply:
ack :)